Explorer/Assets/DCL/Multiplayer/Connections/Archipelago/LiveConnections/AutoReconnectLiveConnection.cs

```diff
@@ -0,0 +1,77 @@
+using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using LiveKit.Internal.FFIClients.Pools.Memory;
+using System;
+using System.Net.Sockets;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.Multiplayer.Connections.Archipelago.LiveConnections
+{
+    public class AutoReconnectLiveConnection : IArchipelagoLiveConnection
+    {
+        private readonly IArchipelagoLiveConnection origin;
+        private readonly Action<string> log;
+        private string? cachedAdapterUrl;
+
+        public bool IsConnected => origin.IsConnected;
+
+        public AutoReconnectLiveConnection(IArchipelagoLiveConnection origin) : this(
+            origin,
+            m => Debug.Log($"{ReportCategory.ARCHIPELAGO_REQUEST}: {m}")
```

Comment:
Shouldn't we use `ReportHub.Log`?

Reply:
I change it to ReportHub.Log but for some reason it won't show the messages if the category is specified as ARCHIPELAGO_REQUEST, but it works if it's UNSPECIFIED